packages/gestalt/src/IconButton.tsx

```diff
@@ -232,6 +275,11 @@ const IconButtonWithForwardRef = forwardRef<HTMLButtonElement, Props>(function I
           size={size}
         />
       </div>
+      {size === 'xl' && (
+        <span className={styles.label} style={{ color: getLabelColor() }}>
```

Comment:
can is just be a costant?

Reply:
I did try it as a constant, but since it relies on 5 different variables I needed to do an array of condition objects that didn't look too good. I'll try thinking of something else, if you have a suggestion I'll be glad to try it out too.

Reply:
Changed it into a variable, I think it looks alright. What do you think?